ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -589,8 +591,19 @@ data NewTipInfo blk = NewTipInfo {
       -- chain being A and having a disconnected C lying around, adding B will
       -- result in A -> B -> C as the new chain. The trigger B /= the new tip
       -- C.
+    , newTipSelectView  :: SelectView (BlockProtocol blk)
+      -- ^ The 'SelectView' of the new tip. It is guaranteed that
+      --
+      -- >>> Just newTipSelectView > oldTipSelectView
+      -- True
+    , oldTipSelectView  :: Maybe (SelectView (BlockProtocol blk))
```

Comment:
This field name does have some friction with various idioms/expectations.

My gut tells me there are two options for the "right" fix.

OPTION 1 rename `NewTipInfo` to something like `SelectionChangeInfo`. I have at least confirmed that the type only occurs in the `AddedToCurrentChain` and `SwitchedToAFork` tracer constructors.

OPTION 2 define a matching `OldTipInfo` and include it as a second argument in both `AddedToCurrentChain` and `SwitchedToAFork`.

I suppose there is a third option:

OPTION 3 rename `oldTipSelectView` to something like `newTipDefeatedSelectView`, but that is pressing the boundaries of the concept of "new tip info".

Reply:
I think I will go with OPTION 1 :+1: